src/index.ts

```diff
@@ -371,6 +371,29 @@ const plugin: JupyterFrontEndPlugin<void> = {
         }
       }
     });
+    // Track user time, and show a reminder to save the notebook every
+    // five minutes using a toast notification.
+    let saveReminderTimeout: number | null = null;
+
+    tracker.widgetAdded.connect((_, panel) => {
+      if (saveReminderTimeout) {
+        window.clearInterval(saveReminderTimeout);
+      }
+
+      panel.context.ready.then(() => {
+        if (panel.context.model.readOnly) {
+          console.log('View-only notebook detected: skipping save reminder.');
+          return;
+        }
```

Comment:
Do we need this? Only the `tracker` has this save reminder timeout attached, not the `readonlyTracker`.